cmd/lncli/commands.go

```diff
@@ -330,16 +331,28 @@ var sendCoinsCommand = cli.Command{
 				"scripts",
 		},
 		coinSelectionStrategyFlag,
+		cli.StringSliceFlag{
+			Name: "utxo",
```

Comment:
If we pass in the same utxo twice the command isn't failing which I think should happen from a UX perspective. I tested(last utxo is a dupe): `reg_zane sendcoins --addr bcrt1pztmhas8h2vg8xfvj4kr4s3kgx69d4usq6uxxmyh6lhslrgllwpns87snxq --utxo 8c787614dc1adfda9d09f5a908853d8e91cc5579bc00e43c3958c5cb7dcfaf46:0 --utxo a2a9f76c008f4fecdcfe34fc93da1a5f4f7e63a1f896dec9869272b101ac89a9:1 --utxo a2a9f76c008f4fecdcfe34fc93da1a5f4f7e63a1f896dec9869272b101ac89a9:1 --sweepall`

Reply:
Nice catch! I would fix it